packages/genesys-spark-components/src/components/stable/gux-pagination-cursor/gux-pagination-cursor.scss

```diff
@@ -45,6 +45,10 @@
     }
   }
 
+  .gux-simple-button-content {
+    display: flex;
+  }
+
```

Comment:
I don't see a big advantage with adding a wrapper element with a class instead of just adding the needed css into the `gux-icon` block

```
  gux-icon {
    display: flex;
    inline-size: var(--gse-ui-button-icon-size);
    block-size: var(--gse-ui-button-icon-size);
  }
```

Reply:
fixed